README.md

```diff
@@ -64,7 +64,7 @@ Now we can call `exact_cover`:
 
     >>> import numpy as np
     >>> import exact_cover as ec
-    >>> S = np.array([[1,0,0,1,0],[1,1,1,0,0],[0,1,1,0,0],[0,0,0,0,1]], dtype=np.int32)
+    >>> S = np.array([[1,0,0,1,0],[1,1,1,0,0],[0,1,1,0,0],[0,0,0,0,1]], dtype=bool)
```

Comment:
(we could import DTYPE_FOR_ARRAY here but for simplicity's sake this is probably better.)